src/applications/bmqstoragetool/m_bmqstoragetool_parameters.h

```diff
@@ -126,6 +130,33 @@ class Parameters {
         ITER* iterator();
     };
 
+    typedef bsl::unordered_map<mqbu::StorageKey, bmqp_ctrlmsg::QueueInfo>
+        QueueKeyToInfoMap;
+    typedef bsl::unordered_map<bsl::string, mqbu::StorageKey> QueueUriToKeyMap;
+
+    class QueueInfo {
```

Comment:
Let's rename this class to avoid conflicts with `bmqp_ctrlmsg::QueueInfo`. Maybe QueueMap or QueueCatalogue?

Reply:
Renamed to QueueMap